week-1-ts-ecosystem/w01e01-typescript-basics/package.json

```diff
@@ -1,5 +1,9 @@
 {
   "name": "w01e01-typescript-basics",
   "version": "1.0.0",
-  "scripts": {}
+  "scripts": {},
```

Comment:
Warto jakiś skrypcik dać na uruchomienie,
jeśli nie `nodemon` + `ts-node` to chociaż samo:

```
"scripts": {
  "start": "ts-node src/main.ts"
},
```